packages/lab/src/dialog/Dialog.tsx

```diff
@@ -55,27 +62,38 @@ export const Dialog = forwardRef<HTMLDivElement, DialogProps>(function Dialog(
   const {
     children,
     className,
-    open = true,
+    open = false,
     onOpenChange,
     status,
-    initialFocus,
+    id,
+    size = "medium",
     ...rest
   } = props;
-  const dialogId = useId() || "dialog";
+  const dialogId = useId(id) ?? "saltDialog";
   const targetWindow = useWindow();
   useComponentCssInjection({
     testId: "salt-dialog",
     css: dialogCss,
     window: targetWindow,
   });
 
+  const currentbreakpoint = useCurrentBreakpoint();
+
   const [showComponent, setShowComponent] = useState(false);
 
-  const { floating, context, getFloatingProps } = useDialog({
+  const { context, floating } = useFloatingUI({
     open,
     onOpenChange,
   });
 
+  const click = useClick(context);
```

Comment:
We should copy how Tooltip does this and call them directly in the useInteractions array